src/libostree/ostree-repo.c

```diff
@@ -3814,7 +3818,8 @@ ostree_repo_equal (OstreeRepo *a,
 OstreeRepoMode
 ostree_repo_get_mode (OstreeRepo  *self)
 {
-  g_return_val_if_fail (self->inited, FALSE);
```

Comment:
I'm somehow surprised no static analyzer flagged this "implicit boolean to OstreeRepoMode coercion".